orangecontrib/text/vectorization/dense.py

```diff
@@ -0,0 +1,164 @@
+from Orange.misc.server_embedder import ServerEmbedderCommunicator
+from orangecontrib.text import Corpus
+
+import zlib
+import base64
+import json
+import sys
+import numpy as np
+import warnings
+
+from typing import List, Tuple, Any, Optional
+
+AGGREGATORS = ['mean', 'sum', 'max', 'min']
+LANGS_TO_ISO = {'English': 'en', 'Slovenian': 'sl', 'German': 'de'}
+LANGUAGES = list(LANGS_TO_ISO.values())
+EMB_DIM = 300
+
+class PretrainedEmbedder:
+    """This class is used for obtaining dense embeddings of documents in 
+    corpus using fastText pretrained models from:
+    
+    E. Grave, P. Bojanowski, P. Gupta, A. Joulin, T. Mikolov, 
+    Learning Word Vectors for 157 Languages.
+    Proceedings of the International Conference on Language Resources and Evaluation, 2018.
+
+    Embedding is performed on server so the internet connection is a 
+    prerequisite for using the class. Currently supported languages are:
+        - English (en)
+        - Slovenian (sl)
+        - German (de)
+
+    Attributes:
+        language (str): ISO 639-1 (two-letter) code of desired language.
+        aggregator (str): Aggregator which creates document embedding (single 
+                          vector) from word embeddings (multiple vectors). 
+                          Allowed values are mean, sum, max, min.
+    """
+
+    def __init__(self, language: str = 'en',
+                 aggregator: str = 'mean') -> None:
+        
+        lang_error = '{} is not a valid language. Allowed values: {}'
+        agg_error = '{} is not a valid aggregator. Allowed values: {}'
+        if(language.lower() not in LANGUAGES):
+            raise ValueError(lang_error.format(language, ', '.join(LANGUAGES)))
+        else:
+            self.language = language.lower() 
+
+        if(aggregator.lower() not in AGGREGATORS):
+            raise ValueError(agg_error.format(aggregator, ', '.join(AGGREGATORS)))
+        else:
+            self.aggregator = aggregator.lower()
+
+        self._dim = EMB_DIM
+        self._embedder = _ServerEmbedder(self.aggregator,
+                                        model_name = self.language,
+                                        max_parallel_requests = 0,
+                                        server_url = '',  
+                                        # TODO set proper url
+                                        embedder_type = 'text') 
+
+    def transform(self, corpus: Corpus, copy: bool = True, 
```

Comment:
Do you think we can replace transform with __call__? It is prefered use of that kind of implementations in orange. It enables calling embedders this way:

```
emb = PretrainedEmbedder(...)
embeddings = emb(corpus)  # __call__
```